us-geological-survey.csl

```diff
@@ -107,6 +107,24 @@
               <text variable="page" suffix=" "/>
               <label variable="page" form="short" plural="never"/>
             </group>
+          <group prefix=" ">
+            <text term="accessed"/>
+          <group prefix=" " suffix=",">
+            <date variable="accessed">
+              <date-part name="month" form="long" suffix=" "/>
+              <date-part name="day" form="numeric" suffix=", "/>
+              <date-part name="year" form="long"/>
+            </date>
+          </group>
+          <group prefix=" ">
+            <text term="at"/>
+            <text variable="container-title" prefix=" " suffix=" "/>
+          </group>
+          <group prefix=" " suffix=".">
+            <text term="at"/>
+            <text variable="URL" prefix=" "/>
+          </group>
+          </group>
           </group>
         </if>
         <else-if type="bill book graphic legal_case legislation motion_picture report song" match="any">
```

Comment:
BTW can you delete `report` here? This case is handeld already above. (I know that this wasn't your change.)